lib/cms_validators.rb

```diff
@@ -5,8 +5,8 @@ module Validate
     require 'health-data-standards'
 
     BASE_DIR = File.expand_path("../../", __FILE__)
-    CMS_EP_CAT1_SCHEMATRON_2016 = 'resources/schematron/cat1/PQRS CMS 2016 QRDA Category I v2.2.sch'
-    CMS_EH_CAT1_SCHEMATRON_2016 = 'resources/schematron/cat1/HQR CMS 2016 QRDA Category 1 v2.1.sch'
+    CMS_EP_CAT1_SCHEMATRON_2016 = 'resources/schematron/cat1/PQRS CMS 2016 QRDA Category I v2.3.sch'
+    CMS_EH_CAT1_SCHEMATRON_2016 = 'resources/schematron/cat1/CMS 2016 QRDA Category 1 - HQR v2.3.sch'
```

Comment:
Yes there is a reason...That is what CMS named the file.  Why did they name the files differently you ask?  Probably oversight.